web/themes/joinup/templates/rdf_entity/rdf-entity--solution--listing-item.html.twig

```diff
@@ -25,10 +25,12 @@
       </div>
     {% endif %}
     <div class="listing__field listing__stats mdl-card__actions">
-      <div class="listing__stat">
-        <span class="listing__text">{{ content.downloads.0 }}</span>
-        <span class="listing__icon icon icon--download"></span>
-      </div>
+      {% if content.downloads.0['#markup'] >= 1  %}
```

Comment:
This add business logic into the theme layer. I'm not very happy. I would suggest that we check here for `content.downloads`:

```
{% if content.downloads  %}
```
And we simply don't add the `#markup` on `solution_rdf_entity_view()` if `$total` equals 0.

Reply:
Fixed!

Reply:
0 also removed!

Reply:
I checked and this is fixed now, thanks!